proposals/0386-package-access-modifier.md

```diff
@@ -266,6 +263,88 @@ Potential solutions include introducing new keywords for specific access combina
 
 Sometimes entire modules are meant to be private to the package that provides them.  Allowing this to be expressed directly would allow these utility modules to be completely hidden outside of the package, avoiding unwanted dependencies on the existence of the module.  It would also allow the build system to automatically namespace the module within the package, reducing the need for [explicit module aliases](https://github.com/apple/swift-evolution/blob/main/proposals/0339-module-aliasing-for-disambiguation.md) when utility modules of different packages share a name (such as `Utility`) or when multiple versions of a package need to be built into the same program.
 
+### Package Boundary Customization
+
+1. Opt-out
```

Comment:
The Language Workgroup had requested this opt-out capability to be included in the proposal, not just discussed in Future Directions.  If you don't think it's a good idea to pursue that in the short term, please let me know why, and I can ask the Workgroup what we think about going forward with the proposal as-is.  But there was a lot of strong feedback in the review that this was important.